### PR TITLE
Remove global reference - Closes #3754

### LINKED ISSUE #3754: Remove global scope references
### Expected behavior
Any of the file within chain module should not be accessing the global scope

### Actual behavior
Still some file are referencing the global scope

### Which version(s) does this affect? (Environment, OS, etc...)
2.2-

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?
There was still reference to global variable or module level scope.

### How did I fix it?
Remove `globals` and scope level `__private`, `modules`, `library`

### How to test it?
run the test app, or run `npm t`

### Review checklist

* The PR resolves #3754 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: framework/src/modules/chain/loader.js [hunk 1]
```diff
@@ -99,47 +98,37 @@ class Loader {
 	 */
 	// eslint-disable-next-line class-methods-use-this
 	isActive() {
```
Comment (pablitovicente) [inline framework/src/modules/chain/loader.js]: Why do we have `isActive()` and `syncing()` that return the same property? I think we should only have `syncing()` that is more clear about what is happening.

File: framework/src/modules/chain/rounds/round.js [hunk 2]
```diff
@@ -300,26 +299,33 @@ class Round {
 		const roundRewards = [...this.scope.roundRewards] || [];
 
 		// Apply exception for round if required
-		if (exceptions.rounds[this.scope.round.toString()]) {
+		if (this.scope.exceptions.rounds[this.scope.round.toString()]) {
```
Comment (pablitovicente) [inline framework/src/modules/chain/rounds/round.js]: Why do we keep `this.scope` in this file only?

File: framework/src/modules/chain/rounds/rounds.js [hunk 3]
```diff
@@ -48,11 +46,14 @@ class Rounds {
 		library = {
```
Comment (pablitovicente) [inline framework/src/modules/chain/rounds/rounds.js]: why do we keep `library` on this file?

File: framework/src/modules/chain/submodules/peers.js [hunk 4]
```diff
@@ -55,7 +55,7 @@ class Peers {
 			channel: scope.channel,
 			blocks: scope.modules.blocks,
 			constants: {
-				minBroadhashConsensus: scope.config.constants.MIN_BROADHASH_CONSENSUS,
+				minBroadhashConsensus: scope.config.constants.minBroadhashConsensus,
```
Comment (pablitovicente) [inline framework/src/modules/chain/submodules/peers.js]: Why do we keep `scope` in this file?

File: framework/src/modules/chain/rounds/round.js (hunk 2 above)
Comment (shuse2) [reply framework/src/modules/chain/rounds/round.js]: I was trying to be minimally invasive for round module (that's why it still has `library` etc)

File: framework/src/modules/chain/rounds/rounds.js (hunk 3 above)
Comment (shuse2) [reply framework/src/modules/chain/rounds/rounds.js]: As commented above

File: framework/src/modules/chain/submodules/peers.js (hunk 4 above)
Comment (shuse2) [reply framework/src/modules/chain/submodules/peers.js]: I didn't want to touch so much because #3806

Comment (pablitovicente) [reply framework/src/modules/chain/submodules/peers.js]: Ok can we create an issue to remove it?

File: framework/src/modules/chain/rounds/round.js (hunk 2 above)
Comment (pablitovicente) [reply framework/src/modules/chain/rounds/round.js]: Can we create an issue to make the refactor in separate PR?

File: framework/src/modules/chain/rounds/rounds.js (hunk 3 above)
Comment (pablitovicente) [reply framework/src/modules/chain/rounds/rounds.js]: Can we create an issue to make the refactor in separate PR?

File: framework/src/modules/chain/submodules/peers.js (hunk 4 above)
Comment (shuse2) [reply framework/src/modules/chain/submodules/peers.js]: I think #3808 should solve this issue 😉

File: framework/src/modules/chain/loader.js [hunk 5]
```diff
@@ -157,301 +135,281 @@ class Loader {
...
 
 	/**
-	 * It assigns components & modules from scope to private constants.
+	 * Loads signatures from network.
+	 * Processes each signature from the network.
 	 *
-	 * @param {components, modules} scope modules & components
-	 * @returns {function} Calling __private.loadBlockChain
+	 * @private
+	 * @param {function} cb
```
Comment (lsilvs) [inline framework/src/modules/chain/loader.js]:
```suggestion
```

File: framework/src/modules/chain/loader.js [hunk 6]
```diff
@@ -157,301 +135,281 @@ class Loader {
...
-	const { data: result } = await library.channel.invoke('network:request', {
-		procedure: 'getSignatures',
-	});
+	/**
+	 * Loads transactions from the network:
+	 * - Validates each transaction from the network and applies a penalty if invalid.
+	 * - Calls processUnconfirmedTransaction for each transaction.
+	 *
+	 * @private
+	 * @param {function} cb
```
Comment (lsilvs) [inline framework/src/modules/chain/loader.js]:
```suggestion
```

File: framework/src/modules/chain/rounds/rounds.js (hunk 3 above)
Comment (shuse2) [reply framework/src/modules/chain/rounds/rounds.js]: This is being rewritten in the BFT branch

File: framework/src/modules/chain/rounds/round.js (hunk 2 above)
Comment (shuse2) [reply framework/src/modules/chain/rounds/round.js]: This is re-written in BFT branch

**Pull request merged**